Makefile

```diff
@@ -85,12 +85,17 @@ python-caches-clean:
 tox-env-clean:
 	rm -rf .tox
 
+lint: check-venv
+	@find rally tests -name "*.py" -exec pylint -j0 -rn --load-plugins pylint_quotes --rcfile=$(CURDIR)/.pylintrc \{\} +
```

Comment:
Shouldn't this be `@find esrally tests ...`? Otherwise it only applies the linter to tests but it should also check production code. I also suggest we add `benchmarks` and `scripts` to the list of directories to check.